src/libs/ReportUtils.ts

```diff
@@ -2193,12 +2191,15 @@ function getReimbursementQueuedActionMessage(reportAction: OnyxEntry<ReportActio
  * Returns the preview message for `REIMBURSEMENT_DEQUEUED` action
  */
 function getReimbursementDeQueuedActionMessage(
-    reportAction: OnyxEntry<ReportActionBase & OriginalMessageReimbursementDequeued>,
+    reportAction: OnyxEntry<ReportAction<typeof CONST.REPORT.ACTIONS.TYPE.REIMBURSEMENT_DEQUEUED>>,
     reportOrID: OnyxEntry<Report> | EmptyObject | string,
     isLHNPreview = false,
 ): string {
+    if (!ReportActionsUtils.isReimbursementDeQueuedAction(reportAction)) {
+        return '';
+    }
```

Comment:
Wondering if this check could cause a similar problem like the one here: https://github.com/Expensify/App/pull/40168/files#r1651736267